renderer/index.html

```diff
@@ -4,52 +4,226 @@
+        }
+
+        .news::-webkit-scrollbar-thumb:hover {
+          background-color: var(--hover-color);
+        }
+
+        .output {
+            width: 90%;
+            min-height: 13px;
+            max-height: 13px;
+            white-space: pre-wrap;
+            overflow-y: hidden; 
+            overflow-x: hidden;
+            margin: 0 auto;
+            margin-bottom: 5px;
+        }
+
+        footer {
+          background-color: var(--primary-color);
+          display: flex;
+          align-content: center;
+          flex-direction: column;
+          position: fixed;
+          bottom: 0;
+          left: 0;
+          right: 0;
+          z-index: 1;
+          padding: 5px 0px 15px 0px;
+        }
+
+        .loading {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            padding: 0 2em;
+        }
+
+
+        progress {
+            width: 80%;
+            height: 40px;
+        }
+
+        button {
+            padding: 10px 20px;
+            font-weight: bold;
+            border: 2px solid var(--input-border-color);
+            border-radius: var(--input-border-radius);
+            background-color: transparent;
+            color: var(--text-color);
+            cursor: not-allowed;
+            transition: background-color 0.3s;
+        }
+
+        button.enabled {
+            cursor: pointer;
+            background-color: var(--accent-color);
+            border-color: var(--accent-color);
+        }
+
+        button.enabled:hover {
+            background-color: #5a6cb8;
         }
     </style>
 </head>
 
 <body>
-    <h1></h1>
-    <progress id="progressBar" value="0" max="100"></progress>
-    <pre id="output"></pre>
-
-    <script>
-        const { ipcRenderer } = require('electron');
-
-        function log(msg) {
-            const out = document.getElementById('output');
-            out.textContent += `\n${msg}`;
-        }
-
-        ipcRenderer.on('sync-progress', (_, msg) => log(msg));
-        ipcRenderer.on('sync-progress-percent', (_, percent) => {
-            document.getElementById('progressBar').value = percent;
-        });
-
-        ipcRenderer.on('auto-start-sync', () => {
-            document.getElementById('output').textContent = '';
-            document.getElementById('progressBar').value = 0;
-            ipcRenderer.invoke('start-sync', { localDir: '' })
-                .then(msg => log(msg))
-                .catch(err => log('Error: ' + err));
-        });
-    </script>
+    <div class="container">
+      <img src="../img/banner2.webp">
+      <div class="news"></div>
+    </div>
+    <footer>
+      <pre id="output" class="output">Preparing update...</pre>
+      <div class="loading">
+        <progress id="progressBar" value="0" max="100"></progress>
+        <button id="playBtn" disabled>Play Game</button>
+      </div> 
+    </footer>
+    
+<script>
```

Comment:
Note for the future (Not for you to change now)

We should move this JS out into something like `/src/uiController.js` and then do 

```suggestion
<script src="src/uiController.js"></script>
```

But we don't need to do this now. I can do this in another PR.